richie/settings.py

```diff
@@ -191,12 +191,12 @@ class Base(DRFMixin, ElasticSearchMixin, Configuration):
         "djangocms_googlemap",
         "djangocms_video",
         "djangocms_picture",
-        # FUN stuff
-        "apps.core",
-        "apps.courses",
-        "apps.persons",
-        "apps.search",
-        "plugins.large_banner",
+        # Richie stuff
+        "richie.apps.core",
+        "richie.apps.courses",
+        "richie.apps.persons",
+        "richie.apps.search",
+        "richie.plugins.large_banner",
```

Comment:
This will be my only comment on this awesome PR.

I think this plugins directory is not worth one more level in our code tree.

Why don't we make it an app so we would have:

- `richie.core`
- `richie.plugins`
- `richie.courses`
- `richie.persons`
- `richie.search`

It is nicer to be able to call `richie.course` instead of `richie.apps.course`.

In `richie.plugins`, we can put all the plugins that come with Richie to help build a learning portal but are not directly related to one of the app (cards, heros, large banners,...). In the other apps we can put the plugins that are related to this app (like @rmoch has already done a person plugin in the `persons` app).

Another option is to put these general plugins in `richie.core`.

WDYT?

Reply:
IMO it's more explicit to put apps in an _ad hoc_ directory. But I have no strong feelings about this design decision.